deploy/migrations/config.js

```diff
@@ -19,15 +21,23 @@ const mainnet = {
     artblocksAddress: "0x6C093Fe8bc59e1e0cAe2Ec10F0B717D3D182056B"
   },
   communityWallet: "0x1cf0df2a5a20cd61d68d4489eebbf85b8d39e18a",
+  deploy_CryptoPunks: false,
+  address_CryptoPunks: "0xb47e3cd837dDF8e4c57F05d70Ab865de6e193BBB",
 }
 const ropsten = {
   communityWallet: "0xe627243104a101ca59a2c629adbcd63a782e837f",
+  deploy_CryptoPunks: true,
+  address_ownerTestCryptoPunks: "0xc66d094ed928f7840a6b0d373c1cd825c97e3c7c",
```

Comment:
what is this address? 0xc66d094ed928f7840a6b0d373c1cd825c97e3c7c

Reply:
ok, add